src/ResourceManager/Network/Commands.Network/packages.config

```diff
@@ -8,7 +8,6 @@
   <package id="Microsoft.Azure.Graph.RBAC" version="3.4.0-preview" targetFramework="net452" />
   <package id="Microsoft.Azure.KeyVault.Core" version="1.0.0" targetFramework="net45" />
   <package id="Microsoft.Azure.Management.Authorization" version="1.0.0" targetFramework="net45" />
-  <package id="Microsoft.Azure.Management.Compute" version="14.0.0-prerelease" targetFramework="net45" />
```

Comment:
I don't have to update this file, but I just found out that Commands.Network project does not reference Microsoft.Azure.Management.Compute, so I just remove it from the packages.config